test-cases/bad-script.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
The rest of the examples have comments showing precisely where the errors are.  Not sure how one would do that for syntax issues, but it would make it clearer what we're expecting to happen here.

Reply:
Good idea! I've added comments. These are basic syntax errors. We may want to update these in the future if we have more specific use cases.